storagenode/monitor/monitor.go

```diff
@@ -138,7 +138,9 @@ func (service *Service) Run(ctx context.Context) (err error) {
 	group.Go(func() error {
 		timeout := service.Config.VerifyDirReadableTimeout
 		return service.VerifyDirReadableLoop.Run(ctx, func(ctx context.Context) error {
+			time0 := time.Now()
```

Comment:
a nitpick but could you call it `startTime` and `endTime` + use something like  `endTime := time.Since(start)`